actions/actions.py

```diff
@@ -691,9 +691,18 @@ def run(self, dispatcher, tracker, domain):
 
         if alg_res and alg_res.get("hits") and len(alg_res.get("hits")) > 0:
             docs_found = True
-            doc_list = algolia.get_algolia_link(alg_res.get("hits"), 0)
+            hits = [
+                hit
+                for hit in alg_res.get("hits")
+                if "Rasa X Changelog " not in hit.get("hierarchy", {}).values()
+                and "Rasa Open Source Change Log "
+                not in hit.get("hierarchy", {}).values()
+            ]
+            if not hits:
+                hits = alg_res.get("hits")
+            doc_list = algolia.get_algolia_link(hits, 0)
```

Comment:
Wasn't sure if this belongs here, or in algolia.py.

Reply:
Hm, i guess it makes sense to keep it here bc it's not algolia specifc, i.e. if we change the names of the rasa changelogs then we have to update it here

Would you mind changing `alg_res` to `algolia_result` here? It took me longer than I should have to interpret what was happening

Reply:
ah sure